### DISCUSSION TITLE
Boa noite ilustres estou com problemas com a validação do saft

### DISCUSSION THREAD

**Discussion by niltonpanzo** [assoft-portugal/SAF-T-AO#203] [Q&A] [answered]
Esse é erro que está sempre mostrar 

1)Existem erros de sintaxe xml no ficheiro (Falha validação xsd)

 Ajudam me por favor

**Comment by niltonpanzo** [comment]
Esse é erro que está sempre mostrar

1)Existem erros de sintaxe xml no ficheiro (Falha validação xsd)

Ajudam me por favor

**Reply by batalhadematos** [reply, marked as answer]
Viva @niltonpanzo,

Com certeza tem problemas na estrutura do ficheiro que gerou e como deve imaginar, pela sua mensagem fica difícil ajudar.

Basicamente o XML deve respeitar uma estrutura e no caso, esta deve estar de acordo com as definições do [XSD](https://github.com/assoft-portugal/SAF-T-AO/blob/master/XSD/SAFTAO1.01_01.xsd).

Existem ferramentas que permitem testar se esta estrutura está correcta e tem alguns posts muito bons aqui no repositório. 

Algumas dessas ferramentas:
- xmllint (linux/mac);
- notepad+ (com um plugin específico - o XML Tools é muito bom) e também roda em windows;

No seu caso, aconselhava olhar primeiro para um exemplo e este [AQUI](https://github.com/assoft-portugal/SAF-T-AO/blob/master/Resources/Examples/SAF-T%20AO%20(PurchaseInvoices).xml) e será um bom começo.

Com certeza vai se familiarizar com a estrutura e a partir daqui avançar... quem sabe para outros erros (o que será bom sinal)

Abraço
